packages/migration/migrations/application-config/20230622074325-update-config-constants.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+
+export const up = async (db, client) => {
+  await db.collection('configs').updateMany(
+    {},
+    {
+      $unset: { HIDE_EVENT_REGISTER_INFORMATION: false },
```

Comment:
```suggestion
      $unset: { HIDE_EVENT_REGISTER_INFORMATION: '' },
```
The value for the field when using $unset doesn't matter so using an empty string is better imo to avoid confusions